### DISCUSSION TITLE
Server mode - updating frequency

### DISCUSSION THREAD

**Discussion by remussamoila** [sergi0g/cup#20] [Q&A] [answered]
Hi!
How do I check the frequency of the update checks in server mode?
Doues it check on a specific interval of time?
Do I have to force update?
Thank you!

**Comment by remussamoila** [comment]
To force check there is a sign in the server webpage:
![image](https://github.com/user-attachments/assets/fbae0b26-3d55-4aee-a6c9-f6930d7ee2af)

Now remains to find what is the usual update time: daily, hourly etc

**Comment by sergi0g** [comment, marked as answer]
The server only checks for updates on launch and when you click on the update icon on the web page. Automation isn't supported for now. The best you can do is have a cron job make a get request to `<SERVER_URL>/refresh` as often as you want to.

**Reply by remussamoila** [reply]
Thank you!
